base/strings/annotated.jl

```diff
@@ -1,5 +1,7 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+const RegionAnnotation = NamedTuple{(:region, :label, :value), Tuple{UnitRange{Int}, Symbol, Any}}
```

Comment:
Thanks for giving this a look. This is mainly to avoid writing `NamedTuple{(:region, :label, :value), Tuple{UnitRange{Int}, Symbol, Any}}` again and again in a bunch of places, but for checking if something `isa` region annotation we would need to use `<:Tuple{., ., <:Any}`.

Maybe this should also be defined as a `const` for convenience? Maybe after doing so the values should be swapped?

I'm not sure, but at least I can say that this is very much a private internal (just with a general name), and so we're not locking ourselves into anything here.

Glad to hear nothing else looks notable, I didn't anticipate any major comments but I do like other people reviewing work I do in `JuliaLang/julia` before it's merged.